CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## Version 6.0.0 -- PENDING
+Upgraded to Angular 14
+
```

Comment:
Can you revert these changes please? We will update the changelog when we cut a release.